### PR TITLE
Implement PUT/DELETE endpoints for Observers.

### PR DESCRIPTION
**Pull request by trupci**
#131

### CODE REVIEW

File: src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs [hunk 1]
```diff
@@ -42,17 +42,59 @@ public async Task<List<ObserverModel>> GetObservers(ObserverListQuery query)
...
+        /// <summary>
+        ///  Adds an observer.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns>Boolean indicating whether or not the observer was added successfully.</returns>
         [Authorize]
         [HttpPost]
         [Route("")]
+        [Produces(type: typeof(bool))]
         public async Task<dynamic> NewObserver(NewObserverModel model)
```
Comment (bvizureanu) [inline src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs]: could this have been modified as following (and then the `ProducesAttribute` wouldn't have been required)? 
```suggestion
        public async Task<bool> NewObserver(NewObserverModel model)
```

File: src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs [hunk 2]
```diff
@@ -42,17 +42,59 @@ public async Task<List<ObserverModel>> GetObservers(ObserverListQuery query)
...
         { }
 
+        /// <summary>
+        ///  Adds an observer.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns>Boolean indicating whether or not the observer was added successfully.</returns>
         [Authorize]
         [HttpPost]
         [Route("")]
```
Comment (bvizureanu) [inline src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs]: is the route attribute required if the template is empty?
```suggestion
```

File: src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs [hunk 3]
```diff
@@ -42,17 +42,59 @@ public async Task<List<ObserverModel>> GetObservers(ObserverListQuery query)
         public void Import(IFormFile file, [FromForm] object a)
         { }
 
+        /// <summary>
+        ///  Adds an observer.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns>Boolean indicating whether or not the observer was added successfully.</returns>
         [Authorize]
```
Comment (bvizureanu) [inline src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs]:
```suggestion
```

the `Authorize` attribute should be implicit on all actions. Was this misbehaving? Did we break anything when refactoring auth? (we specified `AllowAnonymous` for the necessary routes until now)

Comment (trupci) [reply src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs]: Nope, don't know why that was there :)

**Pull request merged**